src/main/java/org/kairosdb/core/datastore/KairosDatastore.java

```diff
@@ -107,6 +110,11 @@ public void setBaseCacheDir(@Named(QUERY_CACHE_DIR) String cacheTempDir) {
         }
     }
 
+    @Inject(optional = true)
+    public void setCacheFilesMetricsProvider(CacheFilesMetricsProvider cacheFilesMetricsProvider) {
```

Comment:
I didn't want to go through all the tests in case of adding a new argument to the constructor. I want to test how it works really, and then clean up the code.
Plus I'm not sure, are we planning to maintain this metric, or it's a temporary solution, that we gonna remove after measurements.